### PR TITLE
Add Alicloud SLS output binding docs

### PR DESCRIPTION
**Pull request by RcXu**
## Description

Add Alicloud SLS output binding docs

## Issue reference

Please reference the issue this PR will close: #[issue  #2709]

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/alicloudsls.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  - name: Endpoint
+    value: "[endpoint]"
+```
+
+## Spec metadata fields
+
+| Field         | Required | Binding support  | Details | Example |
+|---------------|----------|---------|---------|---------|
+| `AccessKeyID`    | Y | Output | Alicloud SLS account access key id. | 
+| `AccessKeySecret` | Y | Output |Alicloud SLS accoun access key secret |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/alicloudsls.md]:
```suggestion
| `AccessKeySecret` | Y | Output | Access key credential secret |
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/alicloudsls.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    value: "[accessKey-secret]"
+  - name: Endpoint
+    value: "[endpoint]"
+```
+
+## Spec metadata fields
+
+| Field         | Required | Binding support  | Details | Example |
+|---------------|----------|---------|---------|---------|
+| `AccessKeyID`    | Y | Output | Alicloud SLS account access key id. |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/alicloudsls.md]:
```suggestion
| `AccessKeyID`    | Y | Output |  Access key ID credential. | 
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/alicloudsls.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+<br />
+
+### Response format
+As Alibaba Cloud SLS producer api is asynchronous mode, so there is no response for this binding and implemented a callback interface to accept the response of success or faild, only record faliure reason to the console log.
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/alicloudsls.md]:
```suggestion
As Alibaba Cloud SLS producer API is asynchronous, there is no response for this binding (there is no callback interface to accept the response of success or failure, only a record for failure any reason to the console log).
```

File: daprdocs/data/components/bindings/alibaba.yaml [hunk 4]
```diff
@@ -22,3 +22,11 @@
   features:
     input: false
     output: true
+- component: Alibaba Cloud SLS
+  link: alicloudsls
+  state: Alpha
+  version: v1
+  since: "1.5"
```
Comment (msfussell) [inline daprdocs/data/components/bindings/alibaba.yaml]:
```suggestion
  since: "1.9"
```

Comment (msfussell) [reply daprdocs/data/components/bindings/alibaba.yaml]: This has been merged into the v1.9 release, correct?

**Comment by msfussell** [comment]
@RcXu - Can you review my comments?

**Comment by RcXu** [comment]
> @RcXu - Can you review my comments?

Thanks very mush!

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**